app/src/main/java/com/android/unio/model/event/Event.kt

```diff
@@ -58,6 +58,20 @@ data class Event(
     val numberOfSaved: Int = 0,
     val eventPictures: ReferenceList<EventUserPicture>,
 ) : UniquelyIdentifiable {
+
+  override fun equals(other: Any?): Boolean {
+    if (this === other) return true
+    if (javaClass != other?.javaClass) return false
+
+    other as Event
+
+    return uid == other.uid
+  }
+
+  override fun hashCode(): Int {
+    return uid.hashCode()
+  }
+
```

Comment:
When do you use those?

Reply:
SonarCloud consider not implementing both equals and hashcode together as a maintanability issue, so that's why I implemented it

Reply:
Ok that makes sense